torch_geometric/nn/norm/batch_norm.py

```diff
@@ -30,18 +30,39 @@ class BatchNorm(torch.nn.Module):
             :obj:`False`, this module does not track such statistics and always
             uses batch statistics in both training and eval modes.
             (default: :obj:`True`)
+        allow_no_batch (bool, optional): If set to :obj:`True`, batches with
+            only a single element will work as though in training mode. That is
+            the running mean and variance will be used.
+            (default: :obj:`False`)
     """
-    def __init__(self, in_channels, eps=1e-5, momentum=0.1, affine=True,
-                 track_running_stats=True):
+    def __init__(self, in_channels: int, eps: float = 1e-5,
+                 momentum: float = 0.1, affine: bool = True,
+                 track_running_stats: bool = True,
+                 allow_no_batch: bool = False):
```

Comment:
```suggestion
                 allow_no_batch: bool = True):
```
Let's make this `True` by default?

Reply:
Personally feel making it False by default might be better, as its a bit of a corner case and doesn't match the pytorch behaviour.